dockerfiles/web/Dockerfile-prod

```diff
@@ -43,8 +43,6 @@ RUN pip3 install -r requirements.txt
 # Packages to install with yarn.
 COPY package.json /usr/src/app/
 RUN yarn
-# Copy python script to initialize database.
-COPY /usr/src/app/
```

Comment:
This deletion addresses the issue in this build: https://github.com/lucyparsons/OpenOversight/actions/runs/5732169896/job/15534595950